pkg/helm/findscore.go

```diff
@@ -0,0 +1,162 @@
+// Copyright 2021 FairwindsOps Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package helm
+
+import (
+	"strings"
+
+	"github.com/fairwindsops/nova/pkg/output"
+	version "github.com/mcuadros/go-version"
+	"helm.sh/helm/v3/pkg/release"
+
+	"k8s.io/klog"
```

Comment:
Nit: The organization of these seems off. I typically see three groups - standard, external, and internal

Reply:
This is what my editor does automatically 🤷